addons/mail/static/src/components/follower/follower.xml

```diff
@@ -4,15 +4,15 @@
     <t t-name="mail.Follower" owl="1">
         <div class="o_Follower">
             <t t-if="follower">
-                <a class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
-                    <img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt="Avatar"/>
+                <a role="menuitem" class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
+                    <img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt=""/>
                     <span class="o_Follower_name" t-esc="follower.name or follower.displayName"/>
                 </a>
                 <t t-if="follower.isEditable">
-                    <button class="btn btn-icon o_Follower_button o_Follower_editButton" title="Edit subscription" t-on-click="_onClickEdit">
+                    <button class="btn btn-icon o_Follower_button o_Follower_editButton" title="Edit subscription" aria-label="Edit subscription" t-on-click="_onClickEdit">
```

Comment:
general remark: it's annoying that we have to duplicate the text in all those cases, it shows something is off. We could possibly store them only once in a field.

Reply:
I kind of a gree, but duplicating aria label with same value as title is something necessary in lot of places. Creating a variable on each of those places won't save lines nor work IMHO.